.buildkite/scripts/bootstrap.sh

```diff
@@ -6,7 +6,17 @@ source .buildkite/scripts/common/util.sh
 source .buildkite/scripts/common/setup_bazel.sh
 
 echo "--- yarn install and bootstrap"
-retry 2 15 yarn kbn bootstrap
+if ! yarn kbn bootstrap; then
+  echo "bootstrap failed, trying again in 15 seconds"
+  sleep 15
+
+  # Most bootstrap failures will result in a problem inside node_modules that does not get fixed on the next bootstrap
+  # So, we should just delete node_modules in between attempts
+  rm -rf node_modules
```

Comment:
We might want to consider `yarn kbn clean` as it also resets things like Bazel if there were an issue there. But deleting `node_modules` should accomplish what we're looking to resolve there without possible side-effects `clean` might have.

Reply:
Though, if we used `clean` we would have to re-extract the plugins. So, nevermind.

Reply:
Yeah, I really hope that there's never a time where Bazel starts to handle failures like yarn does